tests/e2e/controller_test.go

```diff
@@ -251,6 +251,9 @@ func TestOdhOperator(t *testing.T) {
 	mustRun(t, Components.String(), Components.Run)
 	mustRun(t, Services.String(), Services.Run)
 
+	// Run misconfigured CRs test suite
+	mustRun(t, "Misconfigured CRs E2E Tests", misconfiguredCRsTestSuite)
+
```

Comment:
Your tests focus on the DSCI and DSC CRs, so we must run them as part of the `dscManagementTestSuite`.